bindings/python/tests/bindings/test_tokenizer.py

```diff
@@ -493,6 +493,7 @@ def test_splitting(self):
         tokenizer.pre_tokenizer.split = False
         tokenizer.add_tokens([AddedToken("<REPR_END>", rstrip=True, lstrip=True)])
         assert tokenizer.encode("<REPR_END>inform<s>. Hey.       .", add_special_tokens=False).tokens == [
+            "▁",
```

Comment:
This seems like a bug no ?

<REPR_END> is a token on it's own, we shouldn't prepend with `_` no ?